### PR TITLE
Link to plugins list from introduction

### PR DESCRIPTION
**Pull request by scott-ln**
Added a link to the plugins list from the introduction.

### CODE REVIEW

**Comment by snitin315** [comment]
@scott-ln Can you accept CLA?

File: src/content/concepts/plugins.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ contributors:
 
 **Plugins** are the [backbone](https://github.com/webpack/tapable) of webpack. webpack itself is built on the **same plugin system** that you use in your webpack configuration!
 
-They also serve the purpose of doing **anything else** that a [loader](/concepts/loaders) cannot do.
+They also serve the purpose of doing **anything else** that a [loader](/concepts/loaders) cannot do. webpack provides [many such plugins](/plugins/) out of the box.
```
Comment (chenxsan) [inline src/content/concepts/plugins.mdx:16]:
```suggestion
They also serve the purpose of doing **anything else** that a [loader](/concepts/loaders) cannot do. Webpack provides [many such plugins](/plugins/) out of the box.
```

Comment (scott-ln) [reply src/content/concepts/plugins.mdx:16]: Hi @chenxsan, per [the typesetting guide](https://webpack-js-org-git-fork-scott-ln-patch-1-webpack-docs.vercel.app/contribute/writers-guide/#typesetting), "webpack should always be written in lower-case letters. Even at the beginning of a sentence."

@snitin315 Yes, done.

**Comment by scott-ln** [comment]
> @scott-ln Can you accept CLA?

Yes, done.

File: src/content/concepts/plugins.mdx (hunk 1 above)
Comment (scott-ln) [reply src/content/concepts/plugins.mdx:16]: Wait, no. The typesetting guide links to [the media guide](https://github.com/webpack/media#name) which says "Webpack can be written with a capital W at the beginning of a sentence"! Which is it? 😁

Comment (chenxsan) [reply src/content/concepts/plugins.mdx:16]: The one in typesetting guide is outdated. I've file a pull request to fix it https://github.com/webpack/webpack.js.org/pull/5358

**Review by snitin315 — DISMISSED** [review]

File: src/content/concepts/plugins.mdx (hunk 1 above)
Comment (scott-ln) [reply src/content/concepts/plugins.mdx]: Great, thanks. There was another one in the preceding paragraph so I've just changed that too.

**Review by chenxsan — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**